controllers/appgroup_controller.go

```diff
@@ -31,6 +33,7 @@ import (
 const (
```

Comment:
Typically the standard is to move finalizers/annotations related to the CR in the `appgroup_types` file

Reply:
Will move them in a future commit / PR. This would require the ApplicationGroup finalizer to be moved as well.